xgi/drawing/draw.py

```diff
@@ -739,7 +740,30 @@ def _color_arg_to_dict(arg, ids, cmap):
 
     Parameters
     ----------
-    arg : str, dict, iterable, or NodeStat/EdgeStat
+    arg : There are several different valid arguments. They are:
+
+        Single color values
+
+        * str
+        * 3- or 4-tuple
+
+        Iterable of colors
+
+        * numpy array of color values as described above
+        * list of color values as described above
+        * dict of color values as described above
+
+        Iterable of values
+
+        * list of floats
+        * dict of floats
+        * numpy array of floats
```

Comment:
Also works with int?
For brevity and consistency:
```suggestion
        Iterable of numerical values (floats)

        * numpy array
        * list
        * dict
```

Reply:
good catch! yep, I will add that to the docstring.